.github/actions/test_gem/action.yml

```diff
@@ -100,7 +107,7 @@ runs:
             echo "::group::🔎 Appraising ${i}"
             BUNDLE_GEMFILE=gemfiles/${i}.gemfile bundle install --quiet --jobs=3 --retry=4 && \
             BUNDLE_GEMFILE=gemfiles/${i}.gemfile bundle show && \
-            BUNDLE_GEMFILE=gemfiles/${i}.gemfile bundle exec rake test
+            BUNDLE_GEMFILE=gemfiles/${i}.gemfile bundle exec rake test || exit
```

Comment:
would using `bundle install --path` using a relative vendor directory address the permissions issue?

Reply:
Probably! I cannot confirm a fix locally, so I'll throw another commit on here changing the workaround from the separate `chmod` step to including a `--path` switch here. Then I'll push that up and see how CI reacts.

Reply:
`bundle install --path` is deprecated, so I went with `bundle config path` in d7ba5f4. Now watching CI. ⌛

Reply:
Nope. `bundle config path` didn't work for bundler's filesystem permission check. I'm going to revert that commit and go back to `chmod`.

Reply:
@simi any tips here?